### ISSUE TITLE
Condorcet method poll UI issues

### ISSUE THREAD

**Issue by ane-gabriela** [xwikisas/application-xpoll#86]
This issue is related to the new functionality [Condorcet method poll](https://github.com/xwikisas/application-xpoll/issues/76)

Steps to reproduce:

1. Create a new poll with 8/9 proposals, Status "Active" and Type "Allow multiple options in order"

Expected results: A new poll is created with a simple/easy to use UI for the user, keeping the same flow as the other Type options (single and multiple option).

Actual results:

1. The profile picture is overlapping the user name
![NewFunctionality](https://user-images.githubusercontent.com/40691112/140938750-971bc20a-c3ba-4517-9981-991f2331b0d9.jpg)
2. The Recent Polls panel is overlapping some of the selected options
3. Scores are present without any selection -> As per the multiple option selection Scores are 0 until the user makes a selection 
![MultipleChoice](https://user-images.githubusercontent.com/40691112/140940548-fee7a7ad-8450-44c7-a171-14ea301f0d55.jpg)
4. The "Select a value" drop-down list isn't that readable with the color selection
![SelectAValue](https://user-images.githubusercontent.com/40691112/140938556-9d6ad53d-6ccf-4b2b-8cca-19ecad0ad953.jpg)
5. In the Polls Home page all options appear as Winner -> As per the multiple option selection the Winner appears only after a selection is made 
![Winners](https://user-images.githubusercontent.com/40691112/140939999-adb8c1a1-d365-4fe8-ab6e-1f2d5f3c2e7f.jpg)

Note: Also noticed that French translation is missing for "Select a value" 
![FrLanguageSelected](https://user-images.githubusercontent.com/40691112/140950515-644d1cf7-d671-4d0e-9962-6c74c5524d8e.jpg)

Environments: Windows 10, XWiki 13.4.5 with MYSQL 8.0 and IE 11, XPoll Application (Pro) API 2.1

**Comment by trrenty** [comment]
Addressed point one, two and six of the isue.

**With regards to the second point:**

> The Recent Polls panel is overlapping some of the selected options

This is an issue that also affects the other type of polls, namely single option and multiple option. The commit fixes the problem by enabling scrolling if the table overflows and allowing more proposals to fit in before overflowing.
![image](https://user-images.githubusercontent.com/45433221/145415381-fd63b117-e8f5-4e94-8d61-0b2f4fa3c2b0.png)

For a window size of 1920p, 9-10 proposals fit properly given any poll type.

Another solution I implemented was to duplicate the table and evenly disperse the proposals beween each of them. This way, the scrolling wasn't needed but having a poll with many votes and many proposals would create a very long page.
![image](https://user-images.githubusercontent.com/45433221/145417404-a3128af6-a01d-4a69-ac6d-1f8b62221278.png)

A proper solution that would also accomodate mobile devices would require, in my opinion, a complete overhaul of the UI. Displaying the poll with the help of a bar/pie chart would be friendly for both desktops and mobile devices.

![image](https://user-images.githubusercontent.com/45433221/145429844-568151cc-3b2d-413a-a06d-18f6520df3cf.png)

Is the current solution good? Is the proposed solution worth investing in? **WDYT?**

**With regards to the third point:**

> Scores are present without any selection -> As per the multiple option selection Scores are 0 until the user makes a selection

This concerns the way the condorcet winner is calculated. If there are no votes, all the proposals have equal, max scores whereas the multiple option poll is calculated by counting the votes each proposal has got.

**With regards to the fourth point:**

> The "Select a value" drop-down list isn't that readable with the color selection

"Select a value" option is disabled - you can only select a proposal from the dropdown. This being said, I don't think there is a problem with it not readable. **WDYT?**

**With regards to the fifth point:**

> In the Polls Home page all options appear as Winner -> As per the multiple option selection the Winner appears only after a selection is made

This also concerns the way the condorcet winner is calculated. As I said previously, when there are no votes, all proposals have the maximum possible score -> everybody is a winner, thus, the home page will display all the proposals as winners. Having this in mind, should this be handled? **WDYT?**

**Comment by mflorea** [comment]
From what I can see the remaining points are not real problems. The only question is whether we want to work on improving the voting UI for Condorcet polls. For this we would have to do a bit of refactoring so that each poll type can provide a custom display for both edit (vote) and view, in a generic way.

**Comment by mflorea** [comment]
Best is probably to close this issue for 2.1.2 since it already has commits and open another issue for adding support for different display per poll type.

**Comment by mflorea** [comment]
I reported #90 .
